### DISCUSSION TITLE
When octobot cancels an open trade order in dip analyser trading mode?

### DISCUSSION THREAD

**Discussion by mertdogan** [Drakkar-Software/OctoBot#1707] [Q&A]
While my simulation octobot started an order but price lost for a while and now it is upper than order value.

Will strategy automatically cancel this for a while or reason or it will wait for this price for infinity and must i cancel it manually?

Order is at price of 2.57 but...
![image](https://user-images.githubusercontent.com/1324617/118734159-7419c700-b846-11eb-88d1-ccbe346b5ad4.png)
 
...but price is at 2.7 now.

![image](https://user-images.githubusercontent.com/1324617/118734255-b2af8180-b846-11eb-84ba-634a84e58847.png)

**Comment by GuillaumeDSM** [comment]
Hi @mertdogan,

It will cancel it when another buy signal will happen
